doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
++--------------+---------------------------------------------------------------+
+
+5. Plutus Tx constraints
+---------------------------------
+
+`Plutus-tx-constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-tx-constraints>`_ 
+contains a constraints-based API that can be used to generate on-chain validation 
+functions and to build transactions by providing a list of constraints. The main 
+design is to be able to use the same constraints on-chain and off-chain in a Plutus 
+application. The off-chain part generates transactions based on types in cardano-api. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Tx constraints                                                        |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-tx-constraints``                         |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-ledger-api, plutus-script-utils, cardano-api           |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress to support all the functionality from        | 
+|              | plutus-ledger-constraints.                                    |
++--------------+---------------------------------------------------------------+
+
+6. Plutus chain index
+---------------------------------
+
+`Plutus chain index <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-chain-index-core>`_ 
+is an application for indexing useful data from the Cardano blockchain that is to be 
+used in the Contract Monad. The main design is for indexing information related to the 
+UTxO set in order to keep the amount of indexed information proportional to the 
+UTxO set. The component is not meant to index historical information, contrary to 
+the purpose of cardano-db-sync. 
+
++--------------+---------------------------------------------------------------+
+| Plutus chain index                                                           |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-chain-index-core``                       |
+|              | ``plutus-apps/plutus-chain-index``                            |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-script-utils                                           |
++--------------+---------------------------------------------------------------+
+| State        | In the process of being deprecated in favor of Marconi.       | 
++--------------+---------------------------------------------------------------+
+
+7. Contract monad (aka contract API, plutus contract)
```

Comment:
I think the latter names are more accurate. The contract monad may be the main bit we talk about, but it's not the name of the library!

Reply:
Hmm -- good to know. I'll double-check on these names.